permits/forms.py

```diff
@@ -883,6 +883,16 @@ def save(self, permit_request, commit=True):
 class PermitRequestAdditionalInformationForm(forms.ModelForm):
     required_css_class = "required"
 
+    notify_author = forms.BooleanField(
+        label=_("Notifier l'auteur de la demande"), required=False,
+    )
+    reason = forms.CharField(
+        label=_("Raison"),
+        widget=forms.Textarea(attrs={"rows": 1}),
+        required=False,
```

Comment:
Should `reason` be required, but only if status is `STATUS_AWAITING_SUPPLEMENT` ?